api/context_graphql.go

```diff
@@ -0,0 +1,365 @@
+// Go functions that expose the Context-related calls in the GraphQL API.
```

Comment:
I think it will be hard to entirely remove this for a long time, since people may be using the CLI with old versions of server - so I'm hesitant to put this in a TODO right now.